.github/workflows/CI.yml

```diff
@@ -29,41 +29,48 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
+          ref: '1.0.0-alpha1'
       - name: Build OpenSearch
         working-directory: ./OpenSearch
-        run: ./gradlew publishToMavenLocal
+        run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=alpha1 -Dbuild.snapshot=false
 
       # dependencies: common-utils
       - name: Checkout common-utils
         uses: actions/checkout@v2
         with:
-          repository: 'opensearch-project/common-utils'
+          repository: 'dblock/common-utils'
```

Comment:
Can we use `opensearch-project/common-utils`, that will probably work as well.

Reply:
I will switch this once https://github.com/opensearch-project/common-utils/pull/9 is merged.